### PR TITLE
virtual assistant

### PR DESCRIPTION
**Pull request by FdevTech**
i don't now why the first change wasn't merged is there anything wrong?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ A curated list of awesome things and projects built by Algerian developpers.
 - [Cryptili](https://github.com/youben11/cryptili) - Cryptili is a lightweight Android app that allows you to encrypt/decrypt messages and send them through a different app.
 - [Ji3an](https://github.com/Taybou/JiaanApp) - Ji3an is a mobile application for food recipes and cart shopping
 - [OpenClass](https://github.com/youben11/open-class/) - OpenClass is a web app that promote information sharing through organized workshops.
-- [Timgad](https://github.com/OpenDZ/timgad) - Timgad is a kernel Linux Security Module that collects per process and system-wide security protections that are not handled by the core kernel itself
+- [Timgad](https://github.com/OpenDZ/timgad) - Timgad is a kernel Linux Security Module that collects per process and system-wide security protections that are not handled by the core kernel itself.</br>
+-[selma](https://github.com/FdevTech/Selma) - A virtual assistant backed by augmented reality capabilities, it allows you to control different appliances in your home.
```
Comment (youknowriad) [inline README.md]: Like I said in the previous PR. The "selma" repository is still missing a LICENSE file 

and here, you should move "selma" before "Timgad"

Comment (FdevTech) [reply README.md]: have you seen the modification ?

Comment (youknowriad) [reply README.md]: Yes, it's better, there are some typos, but I can fix them. But, the original repository is missing a LICENSE file and I can't merge until it's added.

Comment (FdevTech) [reply README.md]: an MIT LICENSE has been added in the original repo

**Review by youknowriad — APPROVED** [review]
LGTM 👍  Thanks for the updates

**Pull request merged**